### ISSUE TITLE
break in program/video list by seasons

### ISSUE THREAD

**Issue by jfabernathy** [bennettpeter/android-MythTV-Leanfront#91]
New Feature request.

I have multiple TV Recordings and Videos that have multiple seasons/episodes.  The new sort feature fixed the ordering of these by season/episode regardless of the file/title name.

Not sure if this is possible within the constraints of the listing framework, but it would be a convenience if the list was somehow able to be separated by Season. 

As an example I have in my list of Videos all 12 seasons, 278 episodes of The Big Bang Theory which are properly sorted but that is a long list to go thru on one list to get to season 11 episode 10 for example. 

A smaller example is the TV program Yellowstone. I have those currently in my Recorded Programs and they are in order by season/episode, but a long list.

**Comment by bennettpeter** [comment]
My Solution

With Videos, I put each season in a separate directory under a directory for the series.
Monarch of the Glen/Season 01/ Monarch of the Glen/Season 02/ etc.

With recordings, I watch them in order, so I always watch from the front of the list, and delete after I have watched.

If I want to save the recordings I have a script that copies them to an archive directory and converts them to mkv.

My solution will not suit everybody. I will look into how to accommodate this in leanfront.

**Comment by jfabernathy** [comment]
Thanks for your solution on Videos,  which is now my solution.  I created 3 directories underneath Yellowstone called Yellowstone-Season-1, -2, and -3.  I moved the .m4v files into the right directories and now both mythfrontend and Leanfront work much better.  So this new feature is not needed for the Videos category.

As to TV Recordings, I do much the same as you with watch and delete.  For classics I want to put in my Video Library, I run a similar script based off of SendTVtoNAS from the Mythtv Forum.

The only possible need for a change in Recordings is what I have now, which is the Yellowstone first 3 seasons still as MPEG2 files in the Default Group Storage Group.  I'm keeping those until my wife has view them.  I could delete them because they are already compressed and in the Videos storage group, but I think the quality of the original MPEG2 is a little better.

So this is much less a need as I originally thought.  Don't worry about it unless you want another challenge,  :-)
